### ISSUE TITLE
'Cannot read property push of undefined' after finalizing onboarding

### ISSUE THREAD

**Issue by rdkelley** [openeew/openeew-dashboard#126]
![screen_after_add_to_device](https://user-images.githubusercontent.com/1787025/119208230-15c33300-ba56-11eb-8d80-4253e34dc75a.png)

**Comment by krook** [comment]
I've seen this in the context of the `history` object. Is that where you've already explored?

**Comment by rdkelley** [comment]
Yeah, I imagine it has something to do with history but haven't had the chance to troubleshoot it yet. I did refactor some of how history is used in the latest PR, so if we're lucky it might already be fixed in the staging deployment. I'm going to investigate more tomorrow.

**Comment by rdkelley** [comment]
Just tested -- this does not seem to be an issue in the current staging version, so I'm guessing the refactor of the history object in the latest PR fixed it. After we release v.0.8.0, we'll make sure that's accurate and close this issue.

**Comment by krook** [comment]
Sounds good. Even if it's not fixed, I think it's less a functional problem than a simple UI/navigation inconvenience. And if we can't get to the root cause later, maybe we just update error handling to look for it and ignore.

**Comment by krook** [comment]
Have we confirmed whether this has indeed been fixed @rdkelley?
